### PR TITLE
Added WattApp geojson

### PR DESCRIPTION
**Pull request by ThomasRanvikEriksen**
Need to discuss this:

What to do about:
- WattApp Extensions
- "nc": "http://entsoe.eu/ns/nc#"
- The use of @language for string fields (name, description)
- Documentation of CapacitySchedule module

### CODE REVIEW

**Comment by ThomasRanvikEriksen** [comment]
@Sveino 

Have you looked into this?

**Comment by ThomasRanvikEriksen** [comment]
WattApp spesific data and config data should be in own file

File: DIGIN10/Grid/CIMJSON-LD/DIGIN10-30-WattApp-GL.geojson [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+                  "cim:ReactivePower.value": 0.0
+                },
+                "nc:CapacityTimePoint.minAllocatedQ": {
+                  "cim:ReactivePower.value": 0.0
+                }
+              }
+            ]
+          }
+        ],
+        "wattapp:contactEmail": "firmapost@digin.no",
```
Comment (ThomasRanvikEriksen) [inline DIGIN10/Grid/CIMJSON-LD/DIGIN10-30-WattApp-GL.geojson:226]: "schema.org:ContactPoint.email":

File: DIGIN10/Grid/CIMJSON-LD/DIGIN10-30-WattApp-GL.geojson [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+    "properties": "geojson:properties",
+    "type": "@type",
+    "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+    "cim": "http://ucaiug.org/ns/CIM#",
+    "nc": "http://entsoe.eu/ns/nc#",
+    "dcterms": "http://purl.org/dc/terms/",
+    "dcat": "http://www.w3.org/ns/dcat#",
+    "prov": "http://www.w3.org/ns/prov#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "wattapp": "https://wattapp.no"
```
Comment (Sveino) [inline DIGIN10/Grid/CIMJSON-LD/DIGIN10-30-WattApp-GL.geojson:36]: We should use Schedule.org

File: DIGIN10/Grid/CIMJSON-LD/DIGIN10-30-WattApp-GL.geojson [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+  "dcterms:license": {
+    "@id": "https://creativecommons.org/licenses/by-nc-sa/4.0/",
+    "dcterms:title": "CC BY-NC-SA 4.0"
+  },
+  "dcterms:accessRights": {
+    "@id": "http://publications.europa.eu/resource/authority/access-right/PUBLIC",
+    "dcterms:title": "PUBLIC"
+  },
+  "dcat:isVersionOf": {
+    "@id": "https://digin.no/baseprofile/DIGIN10-30-WattApp_GL"
```
Comment (Sveino) [inline DIGIN10/Grid/CIMJSON-LD/DIGIN10-30-WattApp-GL.geojson:82]: Suggest that we refer to Grunnprofil as "Fundation".

File: DIGIN10/Grid/CIMJSON-LD/DIGIN10-30-WattApp-GL.geojson [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+  "dcterms:spatial": {
+    "@id": "https://diginenergi.no/DIGIN10/WattApp"
+  },
+  "dcterms:conformsTo": [
+    {
+      "@id": "http://iec.ch/TC57/ns/CIM/CoreEquipment-EU/3.0"
+    },
+    {
+      "@id": "http://iec.ch/TC57/ns/CIM/CoreEquipment/4.0"
+    }
```
Comment (Sveino) [inline DIGIN10/Grid/CIMJSON-LD/DIGIN10-30-WattApp-GL.geojson:94]: This is not the EQ profile - we should create our own "http://digin.no/ns/CIM/GeoJSON/1.0"

**Review by EmilieSkog — APPROVED** [review]

**Pull request merged**